cfgmgr/teammgr.h

```diff
@@ -29,6 +31,7 @@ class TeamMgr : public Orch
     ProducerStateTable m_appLagTable;
 
     std::set<std::string> m_lagList;
+    std::map<std::string, int> m_lagPIDList;
```

Comment:
Suggest to use `pid_t` as a type for pid.
http://man7.org/linux/man-pages/man2/kill.2.html

Reply:
Updated from int to pid_t